markdown/preface.markdown

```diff
@@ -190,9 +194,8 @@ print >>sys.stderr, "fatal error"  # Python 2
 print("fatal error", file=sys.stderr)  # Python 3
 ```
 
-Ah, that looks a bit more worthwhile.
-What the hell is going on in that Python 2 statement?
-The authors don't rightly know.
+That change certainly seems worthwhile; and what is going on in the
+Python 2 version anyway?  The authors don't rightly know.
```

Comment:
I don't know what Nan was referring to specifically, but I like the sentence. It conveys the idea that it's whacked syntax and deserves its final fate.

btw, since we're picking at nits, @stefanv, could you update "certainly seems worthwhile" back to "seems more worthwhile"? I would argue that it's more useful than just parentheses, but I couldn't argue that the whole transition could be justified by it. I would say the only change needed here is removing "the hell".